symfony/plugins/orangehrmTimePlugin/TimeSheetPeriod/TimesheetPeriod.php

```diff
@@ -17,11 +16,11 @@
  * if not, write to the Free Software Foundation, Inc., 51 Franklin Street, Fifth Floor,
  * Boston, MA  02110-1301, USA
  */
-abstract class TimesheetPeriod {
 
-    abstract public function setTimesheetPeriodAndStartDate($startDay);
+namespace OrangeHRM\Time\TimeSheetPeriod;
```

Comment:
```suggestion
namespace OrangeHRM\Time\TimesheetPeriod;
```
`Timesheet` is single word I guess